lsm.go

```diff
@@ -0,0 +1,352 @@
+package frostdb
```

Comment:
maybe not, but could this be a different package?

Reply:
Yea absolutely will refactor.